server/service/validation_sessions.go

```diff
@@ -1,35 +0,0 @@
-package service
-
-import (
-	"context"
-
-	"github.com/fleetdm/fleet/server/kolide"
-	"github.com/fleetdm/fleet/server/sso"
-	"github.com/pkg/errors"
-)
-
-func (mw validationMiddleware) CallbackSSO(ctx context.Context, auth kolide.Auth) (*kolide.SSOSession, error) {
-	invalid := &invalidArgumentError{}
```

Comment:
This validation was moved into the service method https://github.com/fleetdm/fleet/pull/556/files#diff-00eb3879d14a61155f3f85f7a450987f989256eadd14deb5477ac27f3eb14be0R123